server/raft.go

```diff
@@ -330,6 +338,8 @@ func (s *Server) startRaftNode(cfg *RaftConfig) (RaftNode, error) {
 		c:        s.createInternalSystemClient(),
 		sendq:    sendq,
 		quit:     make(chan struct{}),
+		wtvch:    make(chan struct{}),
+		wpsch:    make(chan struct{}),
```

Comment:
Need to make those at least size 1, no?

Reply:
No always do a select around populating.